releng/make-binary.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# vim: softtabstop=2 shiftwidth=2 expandtab
+
+cleanup() {
+  test -d "${temp}" && rm -rf "${temp}"
+  exit
+}
+
```

Comment:
Should there be a `set -e` here to trigger a hard failure for any uncaught nonzero command return?

Reply:
I'll try that tonight when I test my changes